application/models/Kalkun_model.php

```diff
@@ -40,14 +40,37 @@ function login()
 		$this->db->where('username', $username);
 		$this->db->where('password', $password);
 		$query = $this->db->get();
-		
+
+		$this->session->set_flashdata('bef_login_history_count',
+			$this->session->flashdata('bef_login_history_count')-1);
```

Comment:
How is history count used?

Reply:
Used by PR #203

Reply:
From the PR  #203

> If in the process the POSTed data is lost, the user gets redirected by
using the window.history.go function instead of the referrer
so that the data of the form is not lost.

Reply:
It counts the number of times to go back in the history to reach the referrer page once logged in.
Because in case the user fails to login a few times, the number of pages to go back in the history increases in the browser.